### PR TITLE
Entos: Added records for UHF, DFT, and XTB

### PR DESCRIPTION
**Pull request by sjrl**
Added records for UHF, DFT and XTB. This will be paired with a PR for entos Harness updated ins QCEngine.

### CODE REVIEW

File: qcenginerecords/entos/hf_water_energy/dispatch.in [hunk 1]
```diff
@@ -1,14 +1,10 @@
-dft(
+json_results :=
```
Comment (mattwelborn) [inline qcenginerecords/entos/hf_water_energy/dispatch.in:2]: I was wondering what happened to `name =`.

**Review by mattwelborn — APPROVED** [review]
I'm going to approve this, but with the note that I'm probably going to suggest in your QCEngine PR that you add `entos_policy` to the input generator.

**Comment by sjrl** [comment]
Should I update the `dispatch.in` files to include the `entos_policy` then? It won't affect CI since the `dispatch.in` files are only checked for existence at the moment.

**Comment by mattwelborn** [comment]
I think so. Does `entos_policy` create any output?

**Comment by sjrl** [comment]
Yes it does. I'm happy to go through and update the input and output files.

**Comment by mattwelborn** [comment]
Thanks!

**Comment by sjrl** [comment]
Okay should be good to go! I also made a few changes to `test_validate` to include `dftd3` and `qchem` to the listed programs and updated `Result` to `AtomicResult` and `ResultInput` to `AtomicInput`.

**Comment by mattwelborn** [comment]
You might have meant to post this in the QCEngine PR?

**Comment by sjrl** [comment]
Thanks for the check, but there is a `qcenginerecords/tests/test_validate.py` file that checks to see what programs are listed in QCEngineRecords.

**Pull request merged**